test/query_test.exs

```diff
@@ -0,0 +1,40 @@
+defmodule QueryTest do
+  use ExUnit.Case
+
+  setup_all do
+    Application.put_env(:rethinkdb_ecto_test, TestRepo, [])
+
+    {:ok, conn} = RethinkDB.Connection.start_link
+    {:ok, _} = TestRepo.start_link
+    RethinkDB.Query.table_create("posts") |> RethinkDB.Connection.run(conn)
+
+    {:ok, conn: conn}
+  end
+
+  test "get, insert and update queries work" do
+    {:ok, test_model} = TestRepo.insert(%TestModel{title: "yay"})
+    from_db = TestRepo.get(TestModel, test_model.id)
+    assert test_model == from_db
+
+    # update_changeset = TestRepo.changeset(from_db, %{title: "yayay"})
```

Comment:
I'll create all `CRUD` tests.
The `update` assert is commented because I couldn't create a changeset, I'll take a look on this tomorrow (But if you want to give me a clue will be very welcome!).

Do you want to test the `schema`?
```
test "respect schema" do
    {:error, changeset} = TestRepo.insert(%TestModel{blablabla: "yay"})
    assert changeset.errors == "blablabla"
  end
```
It's not passing, and I don't know how to fix.

And can you authorise the repo on travis?
I copied [.travis.yml](https://github.com/hamiltop/rethinkdb-elixir/blob/master/.travis.yml) from rethink-elixir.